recipes/games101-cgl/all/conanfile.py

```diff
@@ -0,0 +1,95 @@
+from conan import ConanFile
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rmdir, rm
```

Comment:
```suggestion
from conan.tools.files import get, copy, rmdir, rm
```
Remove not needed methods!